src/components/procedure/_cloud-platform-display-tainted-data.mdx

```diff
@@ -0,0 +1,17 @@
+:::info Prerequisite
+- Add rules that use taint mode to your [Rule board](https://semgrep.dev/orgs/-/board). See [Semgrep Registry](https://semgrep.dev/explore) for specific rules. 
```

Comment:
I suggest mentioning p/default first because most users will have that already.

For example:
:::info Prerequisite
Not all Semgrep rules or rulesets make use of taint tracking. Ensure that you have a ruleset, such as `p/default`, that contains rules that use taint tracking. You can also add rules that use taint tracking from Semgrep Registry.
:::

Reply:
Thanks @s-santillan - for consistency, I combined your suggestion with something that we already had in docs - see https://github.com/returntocorp/semgrep-docs/pull/889/commits/6aae4230824b7f21dd8588496c405c03c560da69